src/components/Title/Title.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+class Title extends React.Component {
+    render(){
+        return (
+            <div className="title-container">
+                   <h1 className="Title">Hot Dog</h1>
```

Comment:
I think this should be indented like the h2 below

Reply:
On it